### ISSUE TITLE
Import the project

### ISSUE THREAD

**Issue by roalva1** [chathudan/CountryCodePicker#11]
Hello,

how can I use your project in my Android studio project? do I have to copy all to my project or is there any kind of widget to put in my code?
Regards

**Comment by chathudan** [comment]
See the example

**Comment by roalva1** [comment]
but in the example you have this 

package com.sithagi.countrypicker;

import com.sithagi.countrycodepicker.CountryPicker;
import com.sithagi.countrycodepicker.CountryPickerListener;

so you are accessing to the package itself, so I would have to create a package copying your classes, isn't it? I was wondering if I can create kind of widget and just install it with graddle. 

Thanks a lot!!

**Comment by orrinLife360** [comment]
I agree that a gradle dependency or some release tags would be nice.
